client/src/components/OrderCard/index.tsx

```diff
@@ -28,12 +30,12 @@ export const OrderCard = observer((props: OrderCardProps) => {
 
     return (
         <OrderCardComponent>
-            <img className='orderImg' src={`${process.env.REACT_APP_BASE_URL}${order.img}`} />
+            {width > 768 && <img className="orderImg" src={`${process.env.REACT_APP_BASE_URL}${order.img}`} />}
             <div className='orderInfoContainer'>
                 <div className='orderInfo'>
                     <p className='orderName'>{order.name}</p>
-                    <p>{standardInfo}</p>
-                    <Rating readOnly name="read-only" size='large' value={order.rateInfo.rate} />
+                    <p className='standardInfo'>{standardInfo}</p>
+                    <Rating readOnly name="read-only" size={width > 768 ? 'large' : 'small'} value={order.rateInfo.rate} />
```

Comment:
Maybe creating a separate variable like isDesktop will be better (like const isDesktop = width > 768)